runtime/kusama/src/lib.rs

```diff
@@ -965,7 +972,7 @@ impl pallet_proxy::Config for Runtime {
 pub struct CustomOnRuntimeUpgrade;
 impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
```

Comment:
I just realize that this is not part of Executive, can you add it:
```diff
diff --git a/runtime/polkadot/src/lib.rs b/runtime/polkadot/src/lib.rs
index b3312503c..d33c1898a 100644
--- a/runtime/polkadot/src/lib.rs
+++ b/runtime/polkadot/src/lib.rs
@@ -1086,6 +1086,7 @@ pub type Executive = frame_executive::Executive<
        Runtime,
        AllModules,
        BabeEpochConfigMigrations,
+       CustomOnRuntimeUpgrade,
 >;
 /// The payload being signed in transactions.
 pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
```

Same for polkadot

Reply:
Integration to `frame_executive::Executive` Done :+1: